### ISSUE TITLE
[ServiceBus] Unsupported type 0x40 for array

### ISSUE THREAD

**Issue by vladimir-genkin** [Azure/azure-sdk-for-python#8707]
When connecting to azure service bus topic, reading from it and at the same time renewing the message lock the following exception occurs:

```
Failed to complete mgmt operation.
Status code: 500
Message: b'Unsupported type 0x40 for array. TrackingId:6568100f-6493-4e0e-a8bd-7ca7ba948885_G0, SystemTracker:NoSystemTracker, Timestamp:2019-11-19T12:42:49'
Traceback (most recent call last):
  File "/root/.local/share/virtualenvs/bin-BjCKre66/lib/python3.6/site-packages/azure/servicebus/aio/async_base_handler.py", line 90, in _mgmt_request_response
    callback=callback)
  File "/root/.local/share/virtualenvs/bin-BjCKre66/lib/python3.6/site-packages/uamqp/async_ops/client_async.py", line 337, in mgmt_request_async
    **kwargs))
  File "/root/.local/share/virtualenvs/bin-BjCKre66/lib/python3.6/site-packages/uamqp/async_ops/session_async.py", line 114, in mgmt_request_async
    return parse_response(status, response, description)
  File "/root/.local/share/virtualenvs/bin-BjCKre66/lib/python3.6/site-packages/azure/servicebus/common/mgmt_handlers.py", line 28, in lock_renew_op
    status_code, description, message.get_data()))
azure.servicebus.common.errors.ServiceBusError: Management request returned status code: 500. Description: b'Unsupported type 0x40 for array. TrackingId:6568100f-6493-4e0e-a8bd-7ca7ba948885_G0, SystemTracker:NoSystemTracker, Timestamp:2019-11-19T12:42:49', Data: None

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
...
  File "/root/.local/share/virtualenvs/bin-BjCKre66/lib/python3.6/site-packages/azure/servicebus/aio/async_message.py", line 62, in renew_lock
    expiry = await self._receiver._renew_locks(self.lock_token)  # pylint: disable=protected-access
  File "/root/.local/share/virtualenvs/bin-BjCKre66/lib/python3.6/site-packages/azure/servicebus/aio/async_receive_handler.py", line 163, in _renew_locks
    mgmt_handlers.lock_renew_op)
  File "/root/.local/share/virtualenvs/bin-BjCKre66/lib/python3.6/site-packages/azure/servicebus/aio/async_base_handler.py", line 92, in _mgmt_request_response
    raise ServiceBusError("Management request failed: {}".format(exp), exp)
azure.servicebus.common.errors.ServiceBusError: Management request failed: Management request returned status code: 500. Description: b'Unsupported type 0x40 for array. TrackingId:6568100f-6493-4e0e-a8bd-7ca7ba948885_G0, SystemTracker:NoSystemTracker, Timestamp:2019-11-19T12:42:49', Data: None
```

**Comment by Indy2222** [comment]
This is duplicate of #7336.

**Comment by kaerm** [comment]
@vladimir-genkin thanks for reporting and @Indy2222 thanks for linking the related issue //cc: @annatisch

**Comment by KieranBrantnerMagee** [comment]
Closing this as it's ostensibly fixed by the same fix for #7336, as always if folks are still seeing recurrences of this bug, don't hesitate to reopen this and/or reach out to us with details.

**Comment by AvivAvital2** [comment]
Hi,
Using the documented example [here](https://azuresdkdocs.blob.core.windows.net/$web/python/azure-servicebus/7.0.0b4/azure.servicebus.aio.html#azure.servicebus.aio.ServiceBusReceiver.receive_deferred_messages) and [here](https://github.com/Azure/azure-sdk-for-python/blob/9d218ea0225d7bc4bca86abe96993f9fc6685d92/sdk/servicebus/azure-servicebus/samples/async_samples/receive_deferred_message_queue_async.py) this seems to reproduce on 7.0.0b4.
Are the documented examples are up-to-date for b4?

Running:
azure-common        1.1.25
azure-core               1.6.0
azure-servicebus    7.0.0b4
Python: 3.7.1
